.github/pull_request_template.md

```diff
@@ -1,4 +1,22 @@
+## Why
+
+<!-- Explain why the changes are needed.  -->
+
 Fixes #
 
-Changes proposed in this pull request:
+## What
+
+<!-- Describe changes proposed in this pull request. -->
+
+## Tests
+
+<!-- Describe how the change was tested (both manual and automated) for reviewers to find edge cases more easily. -->
+
+## Checklist
+
+<!-- All items should be verified and marked as done.
+     If an item doesn't apply to the introduced changes - check it as done too. -->
```

Comment:
It is probably about preferences, but if I see checkbox checked I am considering it is done.
I'd prefer to see it strikethrough if not applicable.

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/690/commits/ddb3e9b9b74eb33954d24601e620df4083a86c5c